doc/content/gateways/thethingsindoorgateway/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "The Things Indoor Gateway"
+description: ""
+distributions: ["Community", "Cloud"]
+new_in_version: 3.13.3
+---
+
+This page guides you to connect {{% ttig %}} to {{% tts %}}.
+
+<!--more-->
+
+{{% ttig %}} is an 8 Channel LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/thethingsindoor/). 
+
+{{< figure src="TTIG.jpeg" alt="{{% ttig %}}" >}}
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways and API Keys.
+2. The EUI of the gateway. This EUI is a **16 character** value that can be found at the bottom in the WiFi setup screen.
+{{< figure src="TTIG_EUI.png" alt="{{% ttig %}} EUI" >}}
+
+{{< note >}} The EUI of the gateway is **NOT** the WiFi MAC address printed on the back of the gateway, but is derived from the first number that typically is of the form **58A0CBxxxxxx**. It's located on the top of the sticker below the QR code. For example, if that number is 5BA0CB800BE7, insert **FFFE** after the first 6 characters to make it a 16 character Gateway EUI like 5BA0CBFFFE800BE7{{</ note >}}
+
+3. The WiFi password of the {{% ttig %}}, typically printed on the back of the gateway. 
+
+
+## Claiming {{% ttig %}}
+
+{{% ttig %}} is added to The Things Stack via a process called **Gateway Claiming**.
+
+{{< warning >}} Do not register your TTIGs via the regular option of **Adding Gateways**. {{</ warning >}}
+
+### Procedure
+
+Go to **Gateways** in the top menu, and click **Claim Gateway** to reach the gateway claiming page.
+
+Fill the **Gateway EUI**, the **Claim authentication code** (the WiFi Password), the **Gateway ID** and choose the appropriate **Frequency Plan**.
+
+Click **Claim gateway** to finish.
+
+{{< figure src="TTIG_Claim.png" alt="{{% ttig %}} Claiming" >}}
+
+If your inputs are correct, a new gateway will be created and you will be redirected to the gateway overview page of your newly created gateway.
+
+## Connecting {{% ttig %}}
+
+### New Gateways
```

Comment:
Is this for gateways that haven't been connected before at all? Not even V2? Can you please add a short line on the beginning that explains this